services/src/test/scala/cromwell/pipeline/service/GitLabProjectVersioningTest.scala

```diff
@@ -151,7 +151,7 @@ class GitLabProjectVersioningTest extends AsyncWordSpec with ScalaFutures with M
       "return file with 200 response" taggedAs Service in {
         val project = TestProjectUtils.getDummyProject()
         val path = Paths.get("test.md")
-        val version = dummyVersion;
+        val version = PipelineVersion("version-name")
```

Comment:
Is "version-name" a proper variant of string you use in PipelineVersion? Maybe it would be better to use version-like strings as in all other tests? This thing is quite misleading.